NeoML/include/NeoML/Dnn/Layers/GELULayer.h

```diff
@@ -20,25 +20,54 @@ limitations under the License.
 
 namespace NeoML {
 
-// Activation layer with formual x * Ф(x),
-// where Ф(x) - cumulative distribution function for the normal distribution N(0, 1)
-// This layer uses next approximation: x * sigmoid(1.702 * x)
+// Activation layer with formula x * Ф(x),
+// where Ф(x) - cumulative distribution function of the standard normal distribution N(0, 1)
 class NEOML_API CGELULayer : public CBaseLayer {
 	NEOML_DNN_LAYER( CGELULayer )
 public:
 	explicit CGELULayer( IMathEngine& mathEngine );
 
 	void Serialize( CArchive& archive ) override;
 
+	// CDF can be calculated using the error function (slow) or using an approximation. The precise method is used by default.
+	enum TCalculationMode {
+		// x * 0.5( 1 + erf( x / sqrt(2) ) )
+		CM_Precise,
```

Comment:
The default value must be the same `CM_SigmoidApproximate`.
Otherwise the old code will break somewhere...

Reply:
I need precise GELU in TransformerEncoder layer. Its activation setter doesn't support any flags

Reply:
That should be added/refactor in TransformerEncoder layer. I've already found the examples of using new CGELULayer outside of NeoML, so the default can't be changed...

Reply:
In that case I will do that when this PR is completed (I think about adding `AF_GELUSigmoid = GELU` and `AF_GELUPrecise` to `TActivationFunction`).

Reply:
It is the easiest way I suppose, but should we expect adding AF_GELUTanh later? And some other activations have parameters too... AF enum will grow exponentially(